AirportTicketBooking/Logging/LoginPage.cs

```diff
@@ -0,0 +1,54 @@
+﻿using AirportTicketBooking.Models;
+using AirportTicketBooking.Services;
+
+namespace AirportTicketBooking.Logging;
+
+class LoginPage
+{
+    private static UserService _userService = new UserService();
+
+    public async static void Login()
```

Comment:
We never do `async void`! This is an anti pattern. Be careful https://stackoverflow.com/questions/45447955/why-exactly-is-async-void-considered-bad-practice

```suggestion
    public static async Task Login()
```